custom-metrics-stackdriver-adapter/pkg/adapter/provider/translator.go

```diff
@@ -48,6 +48,9 @@ type Translator struct {
 }
 
 // GetSDReqForPods returns Stackdriver request for query for multiple pods.
+// podList is required to be no longer than 100 items. This is enforced by limitation of "one_of()"
```

Comment:
how about adding such check in the function?

Reply:
Done.